import_users.sh

```diff
@@ -61,8 +61,9 @@ fi
 : ${USERDEL_ARGS:="--force --remove"}
 
 # Initizalize INSTANCE variable
-INSTANCE_ID=$(curl -s http://169.254.169.254/latest/meta-data/instance-id)
-REGION=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/document | grep region | awk -F\" '{print $4}')
+METADATA_TOKEN=$(curl -s -XPUT -H 'x-aws-ec2-metadata-token-ttl-seconds: 60' http://169.254.169.254/latest/api/token)
```

Comment:
Can I get a space between `-X` and `PUT` to keep the style consistent?

Reply:
Hey! Fixed :+1: